website/content/api-docs/secret/azure.mdx

```diff
@@ -238,6 +238,8 @@ information about roles.
 - `application_object_id` (`string: ""`) - Application Object ID for an existing service principal that will
   be used instead of creating dynamic service principals. If present, `azure_roles` will be ignored. See
   [roles docs][roles] for details on role definition.
+- `persist_app` (`bool: ""`) – Specifies where the created app should be persisted between runs of the credentials endpoint.
```

Comment:
Suggesting changing the language slightly. Also, should the default value be `false` here instead of `""`?

```suggestion
- `persist_app` (`bool: false`) – If set to true, persists the created service principal application for the lifetime of the role.```